js/admin-notice.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * Admin Notices for AIOSEOP.
+ *
+ * @summary  Handles the AJAX Actions with AIOSEOP_Notices
+ *
+ * @since    2.4.2
+ * @package all-in-one-seo-pack
+ */
+// phpcs:disable PEAR.Functions.FunctionCallSignature.ContentAfterOpenBracket
+// phpcs:disable PEAR.Functions.FunctionCallSignature.MultipleArguments
+// phpcs:disable PEAR.Functions.FunctionCallSignature.CloseBracketLine
```

Comment:
these can be included in phpcs.xml instead of in the class

Reply:
That's why I added these, so that they can be recognized and added to phpcs; instead of being forgotten. I've had to do the same with the Grunt PR, and a WPCS branch.

When we update the rules, it would also be easier to track which rules are being disabled the most. Which is why I added the rule(s), instead of just using `phpcs:disable` or `@codingStandardsIgnore`.

Reply:
so can we add it to phpc.xml? it is part of this issue